### DISCUSSION TITLE
I'm getting this issue in the google search console, "The certificate chain referenced by the signed exchange 'cert-url' cannot be parsed"

### DISCUSSION THREAD

**Discussion by deprantik** [ampproject/amppackager#490] [Troubleshooting]
According to this, [https://support.google.com/webmasters/answer/7450883#sgx_warning_list](https://support.google.com/webmasters/answer/7450883#sgx_warning_list), I've checked the version, it is up to date.
Can anyone point out what may be the issue here.

**Comment by twifkak** [comment]
We'll need to debug. Would you be willing to share the domain name or example URL with this problem? If you prefer to do so privately, you can DM me on the [AMP Slack](https://github.com/ampproject/amphtml/blob/master/CONTRIBUTING.md#discussion-channels).

If you want to investigate it yourself, you can:
 1. Use `curl -H 'Accept: application/signed-exchange;v=b3' -H 'AMP-Cache-Transform: google;v="1..5"'` to fetch the URL.
 2. Use [`dump-signedexchange`](https://github.com/WICG/webpackage/tree/master/go/signedexchange#dump-a-signed-exchange-file) to find out its corresponding cert-url.
 3. Use `curl` to fetch the cert-url.
 4. Use [`dump-certurl`](https://github.com/WICG/webpackage/blob/master/go/signedexchange/cmd/dump-certurl/main.go) to inspect the contents.

**Reply by deprantik** [reply]
Thanks Devin for the reply. I have sent you a message on amp slack regarding the details you asked. Please let me know if you find something.
